tests/framework/stats/core.py

```diff
@@ -61,9 +62,13 @@ def add_pipe(self, producer: Producer, consumer: Consumer, tag=None):
     def run_exercise(self, fail_fast=False) -> Result:
         """Drive the statistics producers until completion."""
         iterations = self._result['iterations']
+        cpu_model = self._result['custom']['cpu_model_name']
         # This is used for indentation purposes.
 
         for tag, pipe in self._pipes.items():
+            self._logger.info("Running workload {} on processor {}".format(
```

Comment:
this message still won't be visible since we have `log_cli_level = ERROR` in `pytest.ini`.
I don't know what the best solution would be here. What about having this as a `warn` and setting the `log_cli_level=WARN`

Reply:
I think that you can override the log level for a specific logger instance.

Reply:
We can set --log-cli-level=INFO in the pipelines which is what I did.

Reply:
I see, this is fine then